### ISSUE TITLE
Validation Loss is not decreasing - dataset too small?

### ISSUE THREAD

**Issue by Schobs** [MIC-DKFZ/nnUNet#757]

**Comment by FabianIsensee** [comment]
I don't see a problem in the plot. Yes there is some overfitting but the segmentation metric (green line) looks good to me. You would of course get better results (higher scores, less overfitting) with more training cases

**Comment by Schobs** [comment]
Cheers, Fabian. After playing with it some more I can see you're right. I will delete this thread.
